test/db.test.js

```diff
@@ -2252,6 +2253,7 @@ describe('Database', function () {
 
         fs.writeFile(testDb, rawData, 'utf8', function () {
           d.loadDatabase(function (err) {
+            err.should.not.equal(null)
```

Comment:
Ce truc n'a aucun sens : si c'est `null` le should va fail, puisque tu peux pas faire `null.machin`